### ISSUE TITLE
summary api is not a valid json document

### ISSUE THREAD

**Issue by logicminds** [sp-hash/TeamBlackMiner#321]
The data returned from the summary API is not a valid json document as it contain multiple objects instead of just one.  Should be something like the following.  

```json
{ 
"miner":  {
  "name": "TBMiner",
  "version": "1.70",
  "api_version": "1.3",
  "num_gpu_threads": 4,
  "total_hashrate": 390850956,
  "total_accepted": 91,
  "total_rejected": 0,
  "total_stale": 0,
  "uptime_minutes": 19
},
"pool": {
  "url": "us1-etc.ethermine.org",
  "port": 0,
  "worker": "rig1",
  "algo": "etchash",
  "job": "111111",
  "epoch": 270,
  "block": 0,
  "diff": 1,
  "ping": 73.3846153846154
},
"devices": {
  "0": {
    "gpu": 0,
    "pcie_id": 1,
    "board_name": "NVIDIA GeForce RTX 3080",
    "type": "Cuda",
    "xintensity": 284,
    "gpu_temp": 48,
    "mem_temp": 0,
    "fan": 100,
    "core_clock": 1050,
    "mem_clock": 10051,
    "watt": 227,
    "hashrate": 96201365,
    "dag": true,
    "accepted": 24,
    "rejected": 0,
    "stale": 0
  },
  "1": {
    "gpu": 1,
    "pcie_id": 2,
    "board_name": "NVIDIA GeForce RTX 3080",
    "type": "Cuda",
    "xintensity": 284,
    "gpu_temp": 50,
    "mem_temp": 0,
    "fan": 97,
    "core_clock": 1050,
    "mem_clock": 10251,
    "watt": 226,
    "hashrate": 98437321,
    "dag": true,
    "accepted": 22,
    "rejected": 0,
    "stale": 0
  },
  "2": {
    "gpu": 2,
    "pcie_id": 4,
    "board_name": "NVIDIA GeForce RTX 3080",
    "type": "Cuda",
    "xintensity": 284,
    "gpu_temp": 59,
    "mem_temp": 0,
    "fan": 84,
    "core_clock": 1050,
    "mem_clock": 10251,
    "watt": 232,
    "hashrate": 98339204,
    "dag": true,
    "accepted": 20,
    "rejected": 0,
    "stale": 0
  },
  "3": {
    "gpu": 3,
    "pcie_id": 6,
    "board_name": "NVIDIA GeForce RTX 3080",
    "type": "Cuda",
    "xintensity": 284,
    "gpu_temp": 59,
    "mem_temp": 0,
    "fan": 66,
    "core_clock": 1050,
    "mem_clock": 10251,
    "watt": 214,
    "hashrate": 97909899,
    "dag": true,
    "accepted": 25,
    "rejected": 0,
    "stale": 0
  }
}
}
```

**Comment by sp-hash** [comment]
fixed in v1.73, api v3

**Comment by sp-hash** [comment]
Not default in v3 but can be enabled with --api-version 1.4 (beta)
